bigtable/use-cases/fraudDetection/README.md

```diff
@@ -84,4 +84,54 @@ The current Cloud Bigtable instance configuration does not provide any replicati
+It builds the following resources:
+
+| Resource                         | Resource Name                                                                                                                                                                                                 |
+|----------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Cloud Bigtable Instance          | featurestore-{RANDOM\_ID}                                                                                                                                                                                     |
+| Cloud Bigtable Table             | customer-information-{RANDOM\_ID}                                                                                                                                                                             |
+| Cloud Bigtable Column Family     | demographics, history                                                                                                                                                                                         |
+| Cloud Pubsub Input Topic         | transaction-stream-{RANDOM\_ID}                                                                                                                                                                               |
+| Cloud Pubsub Output Topic        | fraud-result-stream-{RANDOM\_ID}                                                                                                                                                                              |
+| Cloud Pubsub Output Subscription | fraud-result-stream-subscription-{RANDOM\_ID}                                                                                                                                                                 |
+| Google Storage Bucket            | fraud-detection-{RANDOM\_ID}                                                                                                                                                                                  |
+| Google Storage Objects           | 1. temp/ directory (for temporary dataflow generated files) <br/> 2. testing_dataset/ directory <br/>3. training_dataset/ directory <br/> 4. ml_model/ directory                                              |
+| VertexAI Model                   | fraud-ml-model-{RANDOM\_ID}                                                                                                                                                                                   |
+| VertexAI Endpoint                | **Name:** fraud-ml-model-ep-{RANDOM\_ID} <br/> **Id:** Determined in runtime, stored in Scripts/ENDPOINT\_ID.output                                                                                           |
```

Comment:
Whats the *Id* part here? Is it explaining what the RANDOM_ID is? If so can we put this cell as follows (drop Name and Id):

fraud-ml-model-ep-{RANDOM\_ID} <br/> _( the RANDOM_ID is determined during runtime and will be stored in `scripts/ENDPOINT_ID.output`)_